persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -29,6 +33,10 @@ object HmdaFileParser {
 
   case class HmdaFileParseState(size: Int = 0, parsingErrors: Seq[List[String]] = Nil) {
     def updated(event: Event): HmdaFileParseState = event match {
+      case TsParsed(ts) =>
```

Comment:
This line could be `case TsParsed(_) | LarParsed(_)` (with the obvious later case removed).

(The above assumes that TS and LAR parse results want to be represented together, not separately. Do they? I haven't thought it through.)